RELEASE_NOTES.md

```diff
@@ -24,78 +24,83 @@ You are strongly advised to take a backup first.
 
 Since darktable 4.2:
 
-- Almost ??? commits to darktable+rawspeed
+- ??? commits to darktable+rawspeed
```

Comment:
FYI I've changed this line because the number of commits provided sometimes seems too precise to use the word "almost". For example release 4.2.0 said "Almost 1411". I would suggest either count the commits precisely (which should be easy enough) or round to the nearest hundred. In all honesty, I would be tempted to remove this metric entirely as the number of commits isn't really a good indication of the size of a change. Sometimes it's just an indication of how many tries it takes to get something right.

Reply:
Noted, I'll either round the numbers or keep the new wording. Thanks a lot for this huge work @elstoc!